src/function/builtin.rs

```diff
@@ -148,6 +148,25 @@ pub fn builtin_function(identifier: &str) -> Option<Function> {
             let result_index = if arguments[0].as_boolean()? { 1 } else { 2 };
             Ok(arguments.swap_remove(result_index))
         })),
+        "some" => Some(Function::new(move |argument| {
+            let arguments = argument.as_tuple()?;
+            if let (Value::Tuple(a), b) = (&arguments[0].clone(), &arguments[1].clone()) {
+                if let Value::Tuple(b) = b {
+                    for item in b {
+                        if a.contains(item) {
+                            return Ok(Value::Boolean(true));
+                        }
+                    }
+                } else {
+                    if a.contains(&b) {
+                        return Ok(Value::Boolean(true));
+                    }
+                }
+                Ok(Value::Boolean(false))
+            } else {
+                Ok(Value::Boolean(false))
```

Comment:
This should be an error. If the first argument is wrong, then `Err(EvalexprError::expected_tuple(wrong_first_value))`, and if the second value is wrong (in the `contains_any` function which expects a tuple as second argument), then `Err(EvalexprError::expected_tuple(wrong_second_value))`.

Reply:
Error handling updated! Please let me know if this new handling is appropriately and I can resolve the comment.